docs/user-guide/vshared-plugin.md

```diff
@@ -0,0 +1,109 @@
+---
+title: vShared volume plugin for Docker
+
+---
+## Overview
+Depending on the underlying block storage device system, it might not be possible to access the same 
+persistent volume across different hosts/nodes simultanously.
+For example, currently users cannot mount the same persistent volume which is created through
+vSphere Docker Volume Service (vDVS) on containers running on two different hosts at the same time.
+
+This can be solved through distributed file systems, such as NFS, Ceph, Gluster, etc.
+However, setting up and maintaining those distributed file systems for docker persistent data usage is not a trivial work. 
+Furthermore, users can face more challenges in order to achieve high availability, scalability, and load balancing. 
+
+__vShared volume plugin for Docker__ provides simultanous persistent volume access between hosts in the
+same Docker Swarm cluster for the base volume plugin service such as vDVS, with zero configuration effort,
+along with high availability, scalability, and load balancing support.
+
+## Detailed documentation
+Detailed documentation can be found on our [GitHub Documentation Page](http://vmware.github.io/docker-volume-vsphere/documentation/).
+
+## Prerequisites
+* Docker version 1.30/17.06.0 is required.
+* To use vShared plugin, hosts must be Docker Swarm mode.
+[How to create a swarm](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/)
+[How to add nodes to the swarm](https://docs.docker.com/engine/swarm/swarm-tutorial/add-nodes/)
+* Base docker volume plugin (e.g. [vSphere Docker Volume Service](https://github.com/vmware/docker-volume-vsphere))
+
+## Usage examples
+
+#### Creating a persistent volume from vShared plugin
+```
+$ docker volume create --driver=vshared --name=SharedVol -o size=10gb
+$ docker volume ls
+$ docker volume inspect SharedVol
+```
+Options for creation will be the same for the base volume plugin.
+Please refer to the base volume plugin for proper options.
+
+#### Mounting this volume to a container running on the first host
+```
+# ssh to node1
+$ docker run --rm -it -v SharedVol:/mnt/myvol --name busybox-on-node1 busybox
+/ # cd /mnt/myvol
+# write data into mounted shared volume
```

Comment:
Change to read/write data. This might give an impression that we can either read or write from one container. Same for line 53.